llama_stack/apis/tools/rag_tool.py

```diff
@@ -87,20 +87,6 @@ class RAGQueryGenerator(Enum):
     custom = "custom"
 
 
-@json_schema_type
-class RAGSearchMode(Enum):
```

Comment:
all you needed to use was perhaps `StrEnum` but your fix is also fine

Reply:
Went with this fix instead so nothing has to be changed later, thanks @ashwinb